apps/bridge/src/components/bridge/Tabs.tsx

```diff
@@ -305,7 +306,8 @@ export default function Tabs({ selectedTab }: { selectedTab: Direction }) {
     }
   }, [hasClaims, hasClosedClaims, updateToast, setHasClosedClaims]);
 
-  const displayFAQ = L1_CHAIN_ID === 1;
+  // const displayFAQ = L1_CHAIN_ID === 1;
+  const displayFAQ = true;
```

Comment:
Is this for debugging?
Did you forget to change it back?

Reply:
Thanks for reminding has been applied